app/views/people/show.erb

```diff
@@ -0,0 +1,18 @@
+<p class='link'><%= link_to "Back to TASK TEAM", "/people/show" %></p>
+
+<h1>More information about <%= "#{@person[:name]}" %></h1>
+<% number = rand(1..6) %>
```

Comment:
This variable is a bit out of place - I would recommend either moving it into the controller action or moving it directing into the image_tag string interpolation since it not being used again later